
## **Metagenomics training @MRC/UVRI and LSHTM**

This training is aimed at introducing the basic insights into quality control, metagenomic NGS data processing, mapping read metagenomics data onto reference genomes, de novo assembly of metagenomics data. As a use-case, we apply this to identifying unique adenoviruses from metagenomics data. The first sessions comprise of lecture materials which introduce metagenomics and its application in biomedical research. These are followed with a series of practical sessions a summary of which is available at [https://alfredug.github.io/adenoMetagenTraining/notes.html](https://alfredug.github.io/adenoMetagenTraining/notes.html).

Register [here](https://docs.google.com/forms/d/1x4La0OaNaTgOCK6cj2FXeNSSdEa5-HAy6LmtiWyTXAA/edit).

**Contact:** Please email [nicholas.bbosa@mrcuganda.org](nicholas.bbosa@mrcuganda.org) for more information.

## **Training Schedule**

### **Day 1: Introduction to metagenomics**

**Instructor(s):** Dr Nicholas Bbosa and Mr Alfred Ssekagiri

| Time  | Session|
| ------------- | ------------- |
|0900 - 0915 |    Welcome and participants introduction|
|1000 - 1100 |    Introduction to metagenomics sequencing|
|1100 - 1130 |    Coffee break|
|1130 - 1230 |    Applications of metagenomics|
|1230 - 1300 |    NGS file formats and Quality control |
|1300 - 1400 |    Lunch |
|1400 - 1500 |    Reference-based mapping and De Novo assembly |
|1500 - 1600 |    Virtual machine installation and set up|

### **Day 2: Introduction to linux commands**

**Instructor(s):** Dr Segun Fatumo, Dr Opeyemi Soremekun and Mr Alfred Ssekagiri

| Time  | Session|
| ------------- | ------------- |
|0900 - 1000 |    Recap|
|1000 - 1100 |    Navigating the linux directory structure |
|1100 - 1130 |    Coffee break|
|1130 - 1230 |    Creating and manipulating files/directories in linux|
|1230 - 1300 |    Inspecting file contents|
|1300 - 1400 |    Lunch |
|1400 - 1500 |    File permissions and ownership |
|1500 - 1600 |    Summary of linux commands |

### **Day 3: Working with metagenomics data - part 1**

**Instructor(s):** Mr Alfred Ssekagiri and Dr Nicholas Bbosa

| Time  | Session|
| ------------- | ------------- |
|0900 - 1000 |    Quality control of metagenomic sequence data|
|1000 - 1100 |    Adapter and quality trimming |
|1100 - 1130 |    Coffee break|
|1130 - 1230 |    Mapping metagenomic reads to reference genomes|
|1230 - 1300 |    Visualising alignment files|
|1300 - 1400 |    Lunch |
|1400 - 1500 |    Generating consensus sequences from alignment maps |
|1500 - 1600 |    Recap of reference based mapping |

### **Day 4: Working with metagenomics data - part 2**

**Instructor(s):** Mr Alfred Ssekagiri and Dr Nicholas Bbosa

| Time  | Session|
| ------------- | ------------- |
|0900 - 1000 |    Removing host reads from metagenomic sequence data|
|1000 - 1100 |    De novo assembly of metagenomic reads - `metaspades`|
|1100 - 1130 |    Coffee break|
|1130 - 1230 |    De novo assembly of metagenomic reads - `megahit`|
|1230 - 1300 |    Gap filling and merging metagenomic assemblies from different assemblers |
|1300 - 1400 |    Lunch |
|1400 - 1500 |    Inspecting contigs from de novo assembly of metagenomes |
|1500 - 1600 |    Recap of de novo metagenomic assembly |

### **Day 5: Wrap up session**

**Instructor(s):** Dr Nicholas Bbosa, Dr Sheila Balinda and Mr Alfred Ssekagiri

| Time  | Session|
| ------------- | ------------- |
|0900 - 1000 |    Batch scripting |
|1000 - 1100 |    Downstream analysis |
|1100 - 1130 |    Coffee break|
|1130 - 1230 |    Wrap-up and feedback|
|1230 - 1300 |    Certificates
|1300 - 1400 |    Lunch |
|1400 - 1500 |    End |
